.github/workflows/weekly.yml

```
@@ -0,0 +1,34 @@
+name: Weekly Updates
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 21 * * SUN"
+
+jobs:
+  weekly:
+    if: github.repository_owner == 'ParkingReformNetwork'
+    runs-on: ubuntu-latest
+    env:
+      ENV_VAR_1: ${{ secrets.INFLUXDB_API_TOKEN }}
```

It should be like this:

```suggestion
      INFLUXDB_API_TOKEN: ${{ secrets.INFLUXDB_API_TOKEN }}
```

Ditto on the the other ones.